app/models/course/assessment/answer.rb

```diff
@@ -57,16 +57,22 @@ class Course::Assessment::Answer < ActiveRecord::Base
   # @param [String|nil] redirect_to_path The path to be redirected after auto grading job was
   #   finished.
   # @param [Boolean] reattempt Whether to create new answer based on current answer after grading.
+  # @param [Boolean] regrade Whether this is a regraded answer that should have a lower priority.
   # @return [Course::Assessment::Answer::AutoGradingJob|nil] The autograding job instance will be
   #   returned if the answer is graded using a job, nil will be returned if answer is graded inline.
   # @raise [IllegalStateError] When the answer has not been submitted.
-  def auto_grade!(redirect_to_path = nil, reattempt = false)
+  def auto_grade!(redirect_to_path: nil, reattempt: false, regrade: false)
```

Comment:
instead of calling the option `regarde`, use a more general name like `low_priority`?   Because all auto generated jobs should set this option, currently we have re-grading job and submission grading job.